File: ims-frontend-master/src/services/GraphService.js
import API from './Api'

const REST_API_BASE_URL = '/graph';


export const listGraphs = () => API.get(REST_API_BASE_URL);

export const listGraphsByCompany = (company) => API.get(REST_API_BASE_URL + "/" + company);

export const createGraph = (graph) => API.post(REST_API_BASE_URL, graph);

export const updateGraph = (id, graph) => API.put(REST_API_BASE_URL + '/' + id, graph);

File: ims-backend-master/src/main/java/com/IMS_Backend/ims_backend/services/GraphService.java
package com.IMS_Backend.ims_backend.services;

import java.util.List;

import com.IMS_Backend.ims_backend.model.Graph;

public interface GraphService {

	Graph createGraph(Graph graph);
	 
	 List<Graph> getAllGraphs();
	 
	 Graph updateGraph(Long id, Graph Employee);
	 
	 List<Graph> findByCompany(String company);
}
